tests/integration/kubernetes/confidential_common.sh

```diff
@@ -174,7 +174,7 @@ function create_coco_pod_yaml() {
 # This function creates pod yaml. Parameters
 # - $1: image reference
 # - $2: annotation `io.katacontainers.config.hypervisor.kernel_params`
-# - $3: anootation `io.katacontainers.config.runtime.cc_init_data`
+# - $3: anootation `io.katacontainers.config.hypervisor.cc_init_data`
```

Comment:
nit: Can we fix the s/anootation/annotation/ typo whilst you are changing this line?